src/coreclr/nativeaot/docs/README.md

```diff
@@ -1,6 +1,5 @@
 # Using Native AOT
 
-- [Limitations](limitations.md)
 - [Compiling applications](compiling.md)
 - [Building native AOT apps in containers](containers.md)
 - [Debugging applications](debugging.md)
```

Comment:
I think we can also delete these lines.

These docs should now be focused on developer docs and advanced, unsupported scenarios, rather than central supported constructs.